python/lsst/ip/diffim/makeKernel.py

```diff
@@ -130,14 +131,28 @@ def run(self, template, science, kernelSources, preconvolved=False):
                 Spatially varying background-matching function.
         """
         kernelCellSet = self._buildCellSet(template.maskedImage, science.maskedImage, kernelSources)
-        templateFwhmPix = evaluateMeanPsfFwhm(template,
-                                              fwhmExposureBuffer=self.config.fwhmExposureBuffer,
-                                              fwhmExposureGrid=self.config.fwhmExposureGrid
-                                              )
-        scienceFwhmPix = evaluateMeanPsfFwhm(science,
-                                             fwhmExposureBuffer=self.config.fwhmExposureBuffer,
-                                             fwhmExposureGrid=self.config.fwhmExposureGrid
+        # Calling getPsfFwhm on template.psf fails on some rare occasions when
+        # the template has no input exposures at the average position of the
+        # stars. So we try getPsfFwhm first on template, and if that fails we
+        # evaluate the PSF on a grid specified by fwhmExposure* fields.
+        # To keep consistent definitions for PSF size on the template and
+        # science images, we use the same method for both.
+        try:
+            templateFwhmPix = getPsfFwhm(template.psf)
+            scienceFwhmPix = getPsfFwhm(science.psf)
+        except InvalidParameterError:
+            self.log.info("Unable to evaluate PSF at the average position. "
+                          "Evaluting PSF on a grid of points."
+                          )
```

Comment:
I don't know if logging this information in many places is helpful. For a given template with a `CoaddPsf` error, this line might end up appearing several times.

Reply:
I think you're right. This message would appear once already from `subtractImages.py`, so I would suggest downgrading the log messages here (and below) to `debug`